ui/src/app/signup/controllers/SignupCtrl.js

```diff
@@ -46,6 +47,42 @@
         ToastService.showToast('Please ensure entered details are correct');
       }
     };
-  };
+
+
+    function passwordValid() {
+      $scope.loginForm.password.$valid = true;
+      $scope.loginForm.password.$dirty = false;
+      $scope.loginForm.password.$invalid = false;
+    }
+    function passwordInValid() {
+      $scope.loginForm.password.$valid = false;
+      $scope.loginForm.password.$invalid = true;
+      $scope.loginForm.password.$dirty = true;
+    }
+
+    $scope.validatePwd = function () {
```

Comment:
```suggestion
    $scope.validatePwd = () => {
```
I think this will fix the codacy issues.